Resources/examples/attributes.yml

```diff
@@ -1,11 +1,13 @@
 data_generator:
-    output_dir: /tmp/generated_data
+    output_dir: app/cache
```

Comment:
No , i homogenize it with JM leroux feature (category generator) to avoid problem of non-created directory specified before